### PR TITLE
Completed: Readme

### PR DESCRIPTION
**Pull request by ashwin2507**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,62 @@
...
+## Analysis
+
+First we will split the data into 60% training set and 40% test set, then perform exploratory data analysis on the training set to assess which features are the most appropriate to train the model. One table will summarize the feature data types and number of missing values, and another will display the summary statistics. These tables will allow us to determine how to clean up and filter the data. A distribution of the average rating target variable will be displayed as a [histogram](https://github.com/UBC-MDS/boardgame_rating_predictor/blob/main/results/rating_distribution.png), and will be used to assess whether the data is imbalanced or skewed. Distributions of the numeric features will be displayed as [histograms](https://github.com/UBC-MDS/boardgame_rating_predictor/blob/main/results/numeric_feature_distribution.png) to show the most common numeric feature values. Additionally, correlations between features will be displayed in a table and plotted as pairwise comparisons to identify features that are strongly correlated, and thus, identifying repetitive features that can be dropped from the training process.
```
Comment (erictsai1208) [inline README.md]: We're doing a 50/50 train test split now to save time.

File: envboard.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  - pip
+  - lightgbm
+  - docopt-ng==0.8.1
+  - requests 2.27.1
+  - numpy==1.23.4
+  - pandas==1.4.4
+  - altair==4.2.0
+  - altair_saver
+  - scikit-learn==1.1.3
+
```
Comment (erictsai1208) [inline envboard.yaml:31]: I think we're also missing vl-convert-python here. We need that package to run the code Joel provided to save the images.

File: README.md [hunk 3]
```diff
@@ -1,29 +1,62 @@
...
 
-## Usage
 
-To reproduce this analysis, clone this repository, install the dependencies listed below, and run the following two commands at the command line/terminal from the root directory of this project:
+## Report
 
+The final report of the project can be found [here]()
+
+## Usage
+  1. Download the dependencies from the .yaml [file]()
```
Comment (erictsai1208) [inline README.md:26]: I believe this is missing the actual link to the yaml file in the brackets.

**Review by xFiveRivers — APPROVED** [review]
I reviewed what Eric said and the changes look good

**Pull request merged**